dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/filter/observation/DefaultDubboClientObservationConvention.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.metrics.filter.observation;
+
+import io.micrometer.common.KeyValues;
+import org.apache.dubbo.common.URL;
+import org.apache.dubbo.rpc.RpcContextAttachment;
+
+import static org.apache.dubbo.metrics.filter.observation.DubboObservation.LowCardinalityKeyNames.NET_PEER_NAME;
+import static org.apache.dubbo.metrics.filter.observation.DubboObservation.LowCardinalityKeyNames.NET_PEER_PORT;
+
+/**
+ * Default implementation of the {@link DubboClientObservationConvention}.
+ */
+public class DefaultDubboClientObservationConvention extends AbstractDefaultDubboObservationConvention implements DubboClientObservationConvention {
+    /**
+     * Singleton instance of {@link DefaultDubboClientObservationConvention}.
+     */
+    public static final DubboClientObservationConvention INSTANCE = new DefaultDubboClientObservationConvention();
+
+    @Override
+    public String getName() {
+        return "rpc.client.duration";
+    }
+
+    @Override
+    public KeyValues getLowCardinalityKeyValues(DubboClientContext context) {
+        KeyValues keyValues = super.getLowCardinalityKeyValues(context.getInvocation());
+        RpcContextAttachment rpcContextAttachment = context.getRpcContextAttachment();
+        URL url = context.getInvoker().getUrl();
```

Comment:
```suggestion
       List<Invoker<?>> invokedInvokers = context.getInvocation().getInvokedInvokers();
      for (Invoker<?> invokedInvoker : invokedInvokers) {
          URL url = invokedInvoker.getUrl();
          // do something you want to
      }
```
Be notice that there may more than one invoker being invoked in one time. E.g. Dubbo support retry for several times when invoke failed.

Reply:
Shouldn't there be multiple observations there, one for each retry?

Reply:
The behavior of retrying is a framework behavior. For our users, it's actually a request, just replayed. 
For example, Dubbo supports automatically requesting another provider when it finds a network exception of one provider. The addresses of the two providers may be completely different. 
Or Dubbo also supports broadcast mode invoke, that is, an actual request is sent to all providers in a manner similar to satellite broadcasting. 
These may cause more than one invoked invoker.

Reply:
I see. So what I did for now is to attach tags only from the first invoker. We could think of adding more tags for additional ones later once there's demand for this, WDYT?

Reply:
Agree.